src/classes/Handsontable.php

```diff
@@ -0,0 +1,830 @@
+    {
+        if (!is_array($this->rowHeights) && !is_int($this->rowHeights)) {
+            return null;
+        }
+        return $this->rowHeights;
+    }
+
+    public function getColWidths()
+    {
+        if (!is_array($this->colWidths) && !is_int($this->colWidths)) {
+            return null;
+        }
+        return $this->colWidths;
+    }
+
+    public function getManualColumnResize()
+    {
+        return $this->manualColumnResize;
+    }
+
+    public function getManualRowResize()
+    {
+        return $this->manualRowResize;
+    }
+
+    public function getSortIndicator()
+    {
+        return $this->sortIndicator;
+    }
+
+    public function setSortIndicator($sortIndicator)
+    {
+        $this->sortIndicator = $sortIndicator;
+    }
+
+    public function setRowHeights($rowHeights)
+    {
+        $this->rowHeights = $rowHeights;
+    }
+
+    public function setColWidths($colWidths)
+    {
+        $this->colWidths = $colWidths;
+    }
+
+    public function setManualColumnResize($manualColumnResize)
+    {
+        $this->manualColumnResize = $manualColumnResize;
+    }
+
+    public function setManualRowResize($manualRowResize)
+    {
+        $this->manualRowResize = $manualRowResize;
+    }
+
+    public function setMinSpareRows($minSpareRows)
+    {
+        $this->minSpareRows = $minSpareRows;
+    }
+
+    public function setDataSchema($dataSchema)
+    {
+        $this->dataSchema = $dataSchema;
+    }
+
+    public function setData($data)
+    {
+        $this->data = $data;
+    }
+
+    public function setContainerName($containerName)
+    {
+        $this->containerName = $containerName;
+    }
+
+    public function setLoadLibrairy($loadLibrairy)
+    {
+        $this->loadLibrairy = $loadLibrairy;
+    }
+
+    public function setLoadDataSource($loadDataSource)
+    {
+        $this->loadDataSource = $loadDataSource;
+    }
+
+    public function setSaveDataSource($saveDataSource)
+    {
+        $this->saveDataSource = $saveDataSource;
+    }
+
+    public function setSave($save)
+    {
+        $this->save = $save;
+    }
+
+    public function setAutosave($autosave)
+    {
+        $this->autosave = $autosave;
+    }
+
+    public function setLoadAction($loadAction)
+    {
+        $this->loadAction = $loadAction;
+    }
+
+    public function setSaveElementId($saveElementId)
+    {
+        $this->saveElementId = $saveElementId;
+    }
+
+    public function setStartRows($startRows)
+    {
+        $this->startRows = $startRows;
+    }
+
+    public function setStartCols($startCols)
+    {
+        $this->startCols = $startCols;
+    }
+
+    public function setWidth($width)
+    {
+        $this->width = $width;
+    }
+
+    public function setHeight($height)
+    {
+        $this->height = $height;
+    }
+
+    public function setRowHeaders($rowHeaders)
+    {
+        $this->rowHeaders = $rowHeaders;
+    }
+
+    public function setColHeaders(array $colHeaders)
```

Comment:
this method is a setter and it returns a value (only in one case). Add documentation to explain why